tag_dedupe/dedupe.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python
+
+import sys
+import subprocess
+import os
+import argparse
+
+AUTH_FILE_PATH_LOCAL = "/auth.json"
+
+def dedupe(input_tag, existing_tags):
+	if input_tag in existing_tags:
+		sys.exit("Tag already exists in remote repository! Exiting build.")
```

Comment:
It would probably be slightly easier to test this if you return a boolean from here, and then sys.exit from the main function.